src/frontend/flight/sync/metrics.rs

```diff
@@ -0,0 +1,95 @@
+use metrics::{
+    counter, describe_counter, describe_gauge, describe_histogram, gauge, histogram,
+    Gauge, Histogram,
+};
+
+const REQUEST: &str = "seafowl_sync_writer_request";
```

Comment:
I think we should rename this `sync` to something else, `seafowl_delta_writer`? `seafowl_writer`? `seafowl_changeset_writer`?

Reply:
I'll go with `seafowl_changeset_writer`.